apps/trading-e2e/src/integration/deposits.feature

```diff
@@ -0,0 +1,113 @@
+Feature: Deposits to vega wallet
+
+  Background:
+    Given I navigate to deposits page
+
+  @todo
+  Scenario: Eth wallet connect message displayed if no ethereum wallet connected
+    Then I can see a message stating no wallet is connected
+    And the connect button is displayed
+
+  @todo
+  Scenario: Cannot deposit if approved amount is 0 (approval amount is 0)
+    And I connect my ethereum wallet
+    And I connect my vega wallet
+    When I set "0" tokens to be approved
+    And I approve the asset tokens
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value    |
+      | To (Vega key) | xxxxxxxx |
+      | Amount        | 50       |
+    And I click to the deposit the funds
+    And I approve the ethereum transaction
+  # The following step is valid and are commented out as currently it cannot be automated
+  # Then I can see the deposit is unsuccessful
+
+  @todo
+  Scenario: Cannot deposit if approved amount is lower than deposit amount
+    When I set "2" tokens to be approved
+    And I approve the asset tokens
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
```

Comment:
Should you not specify what asset you are selecting here?

Reply:
not yet .. we can fill in this blank when we start automating this scenario. we'll need to ensure the asset here is always available to select and we don't know if this would require extra work to set up etc .